packages/apps/desktop-viewer-test/src/frontend/extensions/IModelMergeStatusBarItem.scss

```diff
@@ -2,7 +2,8 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-@import "@itwin/itwinui-css/scss/variables";
+@import '@itwin/itwinui-variables/index.css';
+@import '@itwin/itwinui-css/css/global.css';
```

Comment:
i think you just need to add this global.css import once, at the index.scss at the root

Reply:
so update anytime file that adds this import besides index.scss and remove it

Reply:
done